src/Files.App/ViewModels/UserControls/PreviewPaneViewModel.cs

```diff
@@ -290,13 +290,17 @@ public async Task UpdateSelectedItemPreviewAsync(bool downloadItem = false)
 				{
 					PreviewPaneState = PreviewPaneStates.LoadingPreview;
 
-					if (previewSettingsService.ShowPreviewOnly)
+					if (previewSettingsService.ShowPreviewOnly ||
+						SelectedItem?.PrimaryItemAttribute == StorageItemTypes.Folder)
 					{
 						loadCancellationTokenSource = new CancellationTokenSource();
 						await LoadPreviewControlAsync(loadCancellationTokenSource.Token, downloadItem);
 					}
 					else
+					{
 						await LoadBasicPreviewAsync();
+						return;
```

Comment:
What is this trying to fix? I don't think this fix changes anything.

Reply:
The return wasn't needed in the end, I had to make some other changes in 68cccf089f2357250896a2ffb083fce770344c33.

Reply:
Ok, I missed it.